core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairJobView.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 Telefonaktiebolaget LM Ericsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecchronos.core.repair;
+
+import com.ericsson.bss.cassandra.ecchronos.core.repair.state.RepairStateSnapshot;
+import com.ericsson.bss.cassandra.ecchronos.core.utils.TableReference;
+
+/**
+ * Read only view of a scheduled repair job.
+ */
+public interface RepairJobView
```

Comment:
Food for thought: What benefits do you see having a interface define what variables will be output instead of adding a method into TableRepairJob that returns an instance of a view of itself? I feel that the latter would be a little more understandable for the price of some copying. May be worth investigating how it looks.

Reply:
Good suggestion, I've updated this.